testdata/config.toml

```diff
@@ -0,0 +1,13 @@
+input_file_path = "testdata/sample.json"
```

Comment:
Just my opinion here, but I would not make the input and output files part of the configuration.  I consider configs to be mainly 'static', and orthogonal to inputs and outputs.

Reply:
I'm open to a third opinion here.

Reply:
Currently they are optional in the config file (as well as optional from command line) so there is no need to put them there and use them just from command line. 
Well, I believe that we'll get some day to the point, where crunchy will be run daily by some scripts and I could be easier to run it just passing the config file (if files would be in the same place each run).
But for sure, I'll leave possibility to pass them by hand calling crunchy and setting given by hand overwrites setting read from config file.